Source/Editor/Cooker/Steps/DeployDataStep.cpp

```diff
@@ -214,6 +214,30 @@ bool DeployDataStep::Perform(CookingData& data)
                 FileSystem::NormalizePath(srcDotnet);
                 LOG(Info, "Using .NET Runtime {} at {}", TEXT("Host"), srcDotnet);
 
+                // Get major Version
+                Array<String> pathParts;
+                srcDotnet.Split('/', pathParts);
+                String version;
+                for (int i = 0; i < pathParts.Count(); i++)
+                {
+                    if (pathParts[i] == TEXT("runtimes"))
+                    {
+                        Array<String> versionParts;
+                        pathParts[i - 1].Split('.', versionParts);
+                        const String majorVersion = versionParts[0].TrimTrailing();
+                        uint32 versionNum;
+                        StringUtils::Parse(*majorVersion, majorVersion.Length(), &versionNum);
+                        if (versionNum < GAME_BUILD_DOTNET_RUNTIME_MAX_VER || versionNum > GAME_BUILD_DOTNET_RUNTIME_MIN_VER) // Check for major part
```

Comment:
You could use `Math::IsInRange(versionNum, GAME_BUILD_DOTNET_RUNTIME_MIN_VER, GAME_BUILD_DOTNET_RUNTIME_MAX_VER)` to check for range.